bucket/imagemagick.json

```diff
@@ -6,14 +6,15 @@
     "depends": "ffmpeg",
```

Comment:
Should this be depends or suggest?

Personally I believe everything should move to suggests because users can install dependencies on their own as well

Reply:
According to [this](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=256215) it seems ffmpeg is not a proper dependency:

>Thierry Thomas freebsd_committer 2021-05-28 08:21:51 UTC  
>Actually ffmpeg is used for video delegation when it is found, but this is not a real dependency.

I've only used it for photo editing, so I can't be sure. But since ffmpeg is an extra 100MB maybe it's wise to put it as a 'suggest' with a note to install ffmpeg if the user needs it for video editing - I'll do that.

Reply:
Done that, should be good to go now.